### PR TITLE
deb: Import debian directory used in Ubuntu packages

### PR DESCRIPTION
**Pull request by anselmolsm**
Import the debian dir of Ubuntu packages, currently published
at https://launchpad.net/~anselmolsm/+archive/ubuntu/solettaproject

Signed-off-by: Anselmo L. S. Melo anselmo.melo@intel.com

### CODE REVIEW

File: deb/libsoletta/control [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Maintainer: Anselmo L. S. Melo <anselmo.melo@intel.com>
+Build-Depends: debhelper (>= 8.0.0), cdbs, python3 (>=3.4), chrpath, python3-jsonschema, libudev-dev, libpcre3-dev, libicu-dev, libcurl4-openssl-dev
+Standards-Version: 3.9.4
+Section: libs
+Homepage: http://github.com/solettaproject/soletta
+
+Package: libsoletta-dev
+Section: libdevel
+Architecture: any
+Depends: libsoletta (= ${binary:Version}), python3 (>=3.4)
```
Comment (anselmolsm) [inline deb/libsoletta/control:12]: python3-jsonschema is missing here.

File: deb/libsoletta/control [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Source: libsoletta
+Priority: optional
+Maintainer: Anselmo L. S. Melo <anselmo.melo@intel.com>
+Build-Depends: debhelper (>= 8.0.0), cdbs, python3 (>=3.4), chrpath, python3-jsonschema, libudev-dev, libpcre3-dev, libicu-dev, libcurl4-openssl-dev
```
Comment (anselmolsm) [inline deb/libsoletta/control:4]: libasan0 and libasan1 are missing here. Both are installed together with gcc, so testing the pkgs on Ubuntu/debian with gcc hode this issue.
- only debian has libasan1.

Comment (barbieri) [inline deb/libsoletta/control:4]: We shouldn't compile with sanitizer or use libasan

Em sábado, 12 de setembro de 2015, Anselmo L. S. Melo <
notifications@github.com> escreveu:

> In deb/libsoletta/control
> https://github.com/solettaproject/soletta-packaging/pull/5#discussion_r39339581
> :
> 
> > @@ -0,0 +1,34 @@
> > +Source: libsoletta
> > +Priority: optional
> > +Maintainer: Anselmo L. S. Melo <anselmo.melo@intel.com <javascript:_e(%7B%7D,'cvml','anselmo.melo@intel.com');>>
> > +Build-Depends: debhelper (>= 8.0.0), cdbs, python3 (>=3.4), chrpath, python3-jsonschema, libudev-dev, libpcre3-dev, libicu-dev, libcurl4-openssl-dev
> 
> libasan0 and libasan1 are missing here. both are installed together with
> gcc, so testing the pkgs on Ubuntu/debian with gcc hode this issue.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/solettaproject/soletta-packaging/pull/5/files#r39339581
> .

## 

## Gustavo Sverzut Barbieri

Mobile: +55 (19) 99225-2202
Contact: http://www.gustavobarbieri.com.br/contact

Comment (anselmolsm) [inline deb/libsoletta/control:4]: Makes sense.
We can't continue relying on simply running alldefconfig to generate the .config for packages, fine tuning is needed.

Comment (glima) [inline deb/libsoletta/control:4]: True that, note taken.

**Comment by anselmolsm** [comment]
I forgot about it: The tar.gz generated to start the debian packaging already contained duktape.

**Comment by anselmolsm** [comment]
See v2: #9

**Pull request closed without merging**